core/src/main/java/com/linecorp/armeria/server/docs/MethodInfo.java

```diff
@@ -98,6 +98,27 @@ public MethodInfo(String serviceName, String name,
         );
     }
 
+    /**
+     * Creates a new instance.
+     */
+    public MethodInfo(String serviceName, String name,
```

Comment:
I have added yet more craziness to this file. Hopefully not for this PR, but probably need to get a builder in